docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc

```diff
@@ -830,3 +831,129 @@ Error: error loading agent config: error loading raw config: fail to read config
+----
+
+The previous generates a local file named `elastic_agent_installation_complete_manifest.yaml`, which you can use for further analysis. It contains the complete set of resources required for the {agent} installation, including:
+
+* RBAC objects (`ServiceAccounts`, `Roles`, etc. )
+
+* `ConfigMaps` and `Secrets` for {agent} configuration.
+
+* {agent} Standalone deployed as a `DaemonSet`
+
+* https://github.com/kubernetes/kube-state-metrics[Kube-state-metrics] deployed as a `Deployment`.
+
+The content of this file is equivalent to the one obtained when following the "<<running-on-kubernetes-standalone>>" document, with the exception of `kube-state-metrics`, which is not included in the other method.
+
+Possible issues:
+
+* If your user don't have *cluster-admin* privileges, the RBAC resources creation might fail.
+
+* Some Kubernetes security mechanisms (like https://kubernetes.io/docs/concepts/security/pod-security-standards/[Pod Security Standards]) could make part of the manifest to be rejected, as `hostNetwork` access and `hostPath` volumes are required.
+
+* If you already have an installation of `kube-state-metrics`, it could make part of the manifest installation to fail or to update your existing resources wihtout notice.
+
+[discrete]
+[[agent-kustomize-after]]
+=== Failures occurring within specific components after installation
+
+If the installation is correct, all resources are deployed, but data is not flowing as expected (for example you don't see anything on *[Metrics Kubernetes] Cluster Overview* dashboard), check the following items:
+
+. Check resources status and ensure they are all `Running`
++
+[source,sh]
+----
+kubectl get pods -n kube-system | grep elastic
+kubectl get pods -n kube-system | grep kube-state-metrics
+----
+
+. Describe the Pods in case they are in `Pending` state:
++
+[source,sh]
+----
+kubectl describe -n kube-system <name_of_elastic_agent_pod>
+----
+
+. Check logs of elastic-agents and kube-state-metrics, and look for errors:
++
+[source,sh]
+----
+kubectl logs -n kube-system <name_of_elastic_agent_pod> 
+kubectl logs -n kube-system <name_of_elastic_agent_pod> | grep -i error
+----
++
+[source,sh]
+----
+kubectl logs -n kube-system <name_of_kube-state-metrics_pod> 
+----
+
+Possible issues:
+
+* Connectivity, authorization, or authentication issues when connecting to Elasticsearch:
++
+Ensure the API Key and Elasticsearch destination endpoint used during the installation is correct and is reachable from within the Pods.
++
+In an already installed system, the API Key is stored in a `Secret` named `elastic-agent-creds-<hash>`, and the endpoint is configured in the `ConfigMap` `elastic-agent-configs-<hash>`.
+
+* Only missing cluster-level metrics (provided by `kube-state-metrics`):
```

Comment:
I want to describe a common issue where node level metrics are flowing but cluster level metrics are failing (only these ones). Probably I will just remove the `Only`, because even if all metrics are failing, the troubleshooting details for cluster-level ones would still be valid (although when all fails there are more chances for the cause to be at permissions, endpoint or connectivity issues)

Reply:
resolving, as I've prepared a change related with this.